### PR TITLE
fix(ui): scorecards button layout and new colors

### LINKED ISSUE #61: Update scorecards button and icon colors
**Problem**
After switching to tabler, the colors remained the same

**Solution**
[Mockup](https://www.figma.com/design/kQOH4s5DnfWL0p6o6tlMQz/Draft-%5BATS%5D?node-id=1283-6726&t=2SADNKfvpSkpYFeW-4)

Update the colors of the scores buttons:
- Irrelevant - background-color: red-500, border-color: red-500
- Might be irrelevant - background-color: red-300, border-color: red-500
- Good candidate - background-color: green-300, border-color: green-500
- Perfect candidate - background-color: green-500, border-color: green-500

This is how the buttons should look without the option selected:
image

And this is the active state:
image

---

[Mockup](https://www.figma.com/design/kQOH4s5DnfWL0p6o6tlMQz/Draft-%5BATS%5D?node-id=1171-3878&t=2SADNKfvpSkpYFeW-4)
And for the face icons in the show card:
- mood_sad - red-500
- mood_empty - red-300
- mood_smile - green-300
- mood_happy - green-500

---

Also please fix icon placement in Score buttons: icon should be above text
image

### PR DESCRIPTION
**Pull request by ksnitsky**
Closes #61.

- [x] I have reviewed my code in "Files changed" tab.
- [x] I have re-read the issue and this PR fully resolves it.

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -108,8 +108,8 @@ en:
     address: Address
     advancement:
       good_candidate: Good candidate
-      irrelevant: Completely irrelevant
-      might_be_irrelevant: Might be irrelevant
+      irrelevant_candidate: Completely irrelevant
+      relevant_candidate: Might be irrelevant
```
Comment (wrmk) [inline config/locales/en.yml:112]: looks strange, may we rename `relevant_candidate` to `might_be_irrelevant_candidate` or smth like this?

File: app/views/ats/scorecards/_form.html.slim [hunk 2]
```diff
@@ -90,41 +90,33 @@ ruby:
...
-                          placeholder: "",
-                          value: scorecard.summary
+  .card
+    .card-header
+      span.card-title Summary
+    .card-body
+      .d-flex
+        label.d-flex.form-label data-bs-toggle="tooltip" data-bs-placement="top" title="This is required question" Score
+      .form-selectgroup
+        - [%i[irrelevant mood_sad],
```
Comment (wrmk) [inline app/views/ats/scorecards/_form.html.slim:100]: may we compose one constant with score, icon and color and reuse it here and in scorecards helper?

File: app/views/ats/scorecards/_form.html.slim [hunk 3]
```diff
@@ -90,41 +90,33 @@ ruby:
                               rows: 2,
                               placeholder: ""
 
-  .profile-card
```
Comment (wrmk) [inline app/views/ats/scorecards/_form.html.slim:93]: why do we decided to remove this?

File: config/locales/en.yml (hunk 1 above)
Comment (ksnitsky) [reply config/locales/en.yml:112]: In the database, this score is named relevant

File: app/views/ats/scorecards/_form.html.slim (hunk 3 above)
Comment (ksnitsky) [reply app/views/ats/scorecards/_form.html.slim:93]: This class didn't do anything

**Review by wrmk — APPROVED** [review]

**Pull request merged**